CMakeLists.txt

```diff
@@ -9,7 +9,26 @@ set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
-option(ENABLE_SSE2 "Build with SSE2 CPU instructions" ON)
+include(CheckCXXCompilerFlag)
+include(CMakePushCheckState)
+
+# Note that recent versions of MSVC default to producing SSE2 instructions for x86
```

Comment:
Just for curiosity's sake, do you happen to know which "recent versions" this applies to?

Reply:
According to some Googling, it appears that SSE2 became the default in VS 2015.